data_diff/diff_tables.py

```diff
@@ -420,7 +426,7 @@ def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_coun
             return
 
         if level == 1:
-            self.stats["table1_count"] = self.stats.get("table1_count", 0) + count1
+            self.stats["table_count"] = self.stats.get("table_count", 0) + max(count1, count2)
```

Comment:
Maybe we should count both tables?

Reply:
You mean independently? And then `max` them in the CLI? Sure, I don't really care since it's the only user. I guess since it's exposed via the API we can do that

Reply:
Done